apps/demo/app/flexible-editor/page.tsx

```diff
@@ -0,0 +1,42 @@
+'use server'
```

Comment:
Est-ce qu'il y a une raison d'utiliser use server ici? habituellement c'est utilisé pour les [server actions](https://nextjs.org/docs/app/building-your-application/data-fetching/server-actions-and-mutations), donc pour les fonctions et non les composants

Si c'est pour du code qui doit etre seulement sur le serveur, plutot utiliser ['server-only'](https://nextjs.org/docs/app/building-your-application/rendering/composition-patterns#keeping-server-only-code-out-of-the-client-environment)